pyproject.toml

```diff
@@ -143,6 +143,8 @@ dependencies = [
   "httpx", # For OpenAI testing
   "respx", # For OpenAI testing
   "nest-asyncio", # for executor testing
+  "astunparse; python_version<'3.9'",
+  "sqlean.py",
```

Comment:
Are we planning on adding `sqlean` as a stopgap until we figure out the conda situation?

Reply:
yea, otherwise the unit tests won't pass on GitHub CI

Reply:
Did you not add this to our main package dependencies because we might remove the dependency?